pkg/webhooks/webhooks.go

```diff
@@ -0,0 +1,25 @@
+package webhooks
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	trainingoperator "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
+	"github.com/kubeflow/training-operator/pkg/webhooks/pytorch"
+)
+
+type WebhookSetupFunc func(manager manager.Manager) error
+
+var (
+	SupportedSchemeWebhook = map[string]WebhookSetupFunc{
+		trainingoperator.PyTorchJobKind: pytorch.SetupWebhook,
+		trainingoperator.TFJobKind:      scaffold,
+		trainingoperator.MXJobKind:      scaffold,
+		trainingoperator.XGBoostJobKind: scaffold,
+		trainingoperator.MPIJobKind:     scaffold,
+		trainingoperator.PaddleJobKind:  scaffold,
```

Comment:
I prefer to implement these webhooks in other PRs so that we can keep this PR minimal required.